### PR TITLE
Release 2.2 - Improved Fluent DSL, Automatic Step on Failure and Last Step and Documentation

### PR DESCRIPTION
**Pull request by bruderol**
maybe somebody can review and merge this changes that I did for release 2.2 ?

@dola @xeronimus ??

I think now only the documentation could need some improvements and the rest should be okay to be released.

### CODE REVIEW

**Comment by dola** [comment]
I will have a look next week. Most likely on Thursday (Ascension day)

File: example/test/exampleFluentDslPendingUseCase.spec.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/**
+ * Example for a use case that is currently set to pending
+ */
+useCase('Example Pending Use Case with Fluent DSL')
```
Comment (bruderol) [inline example/test/exampleFluentDslPendingUseCase.spec.js]: I do not like this, because bla bla ....

**Comment by bruderol** [comment]
@dola Please do not delete the release branch anymore! This branch should live until we have finished this release! And currently some projects are using the library through this branch to fetch the library release candidate for testing. so please do not remove, this branch is for all remaining work for the release! Thank you!

**Comment by dola** [comment]
Sorry. I did not realize it was the release branch when I deleted it when merging the PR on github.

**Pull request merged**